app/services/submissions/search_service.rb

```diff
@@ -20,6 +20,16 @@ def call
 
   private
 
+    def sanitize_query(query)
+      return nil if query.blank?
+
+      query
+        .gsub(/[()]/, "")
```

Comment:
The initial issue seemed to be to do with a `&` in the query string, and that still seems to cause an error. Should we not guard against that as well? 

And rather than do this piecemeal for every naughty character we can think of, I wonder if it would make more sense to try to do something more comprehensive. https://stackoverflow.com/questions/14103880/escaping-special-characters-in-to-tsquery suggests `plainto_tsquery` or `$$` syntax?

Reply:
So there's a couple of issues, one was unmatched parens, another was unescaped ampersands (which I'm looking at still, think I have it just adding another test case)

`plainto_tsquery` was a good find though, that seems to resolve it.

> can we also think about changing content? Whether we need to let the user know what the reason for the error is?

So all that we get back error-wise is "something is wrong with this query", we don't know exactly what; but using `plainto_tsquery` now it seems like nothing will cause it to break.

Reply:
The problem we have now though is anything the user enters we'll accept, so if they do add in an invalid character we don't have any way of discerning whether it should be part of the search query or not.

Reply:
Discussed internally and decided this is fine, and to ticket up how we handle invalid searches